packages/odf/modals/s3-browser/delete-and-empty-bucket/EmptyBucketModal.tsx

```diff
@@ -0,0 +1,275 @@
+import * as React from 'react';
+import { DeleteObjectsCommandOutput } from '@aws-sdk/client-s3';
+import { NoobaaS3Context } from '@odf/core/components/s3-browser/noobaa-context';
+import { CommonModalProps } from '@odf/shared/modals';
+import { S3Commands } from '@odf/shared/s3';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { useModal } from '@openshift-console/dynamic-plugin-sdk';
+import { TFunction } from 'i18next';
+import { Trans, useTranslation } from 'react-i18next';
+import {
+  Modal,
+  Button,
+  Text,
+  TextInput,
+  ModalVariant,
+  ValidatedOptions,
+  TextInputTypes,
+  TextVariants,
+  TextContent,
+  Alert,
+  AlertVariant,
+  AlertActionCloseButton,
+  AlertActionLink,
+  FormGroup,
+} from '@patternfly/react-core';
+import { LazyDeleteBucketModal } from './lazy-delete-and-empty-bucket';
+
+type EmptyBucketModalProps = {
+  bucketName: string;
+  noobaaS3: S3Commands;
+  refreshTokens?: () => void;
+  setEmptyBucketResponse: React.Dispatch<
+    React.SetStateAction<EmptyBucketResponse>
+  >;
+};
+
+export const getTextInputLabel = (t: TFunction, bucketName: string) => (
+  <Trans t={t as any} ns="plugin__odf-console">
```

Comment:
```suggestion
  <Trans t={t}>
```
Maybe the type function import of TFunction is wrong. Refer to other files.